objects/LuaScript.ttslua

```diff
@@ -1898,15 +1919,26 @@ function setupStartingResources()
 end
 
 function menuStart(_, value, _)
-  -- Only show the manual choice menu if player chose that during setup options
   local menuChoice = tonumber(value)
-  if menuChoice == 3 and checkToggle('leaders4') then
+  --If the player chose campaign set a variety of options then show the manual leader choosing UI
+  if menuChoice == 1 then
+    uiOff()
```

Comment:
This is called twice (here and in setup)

Reply:
Can you clarify what you mean by this? Are you talking about the uiOff()?

Reply:
Yeah exactly

Reply:
Ah wait is that because of the intermediate UI for the Rival?

Reply:
It’s because with the campaign you need a leader so I automatically display the leader selection UI. To close that you need the uioff